traitsui/qt4/clipboard.py

```diff
@@ -1,3 +1,13 @@
+# (C) Copyright 2008-2021 Enthought, Inc., Austin, TX
```

Comment:
I'm assuming it was intentional that many of these are left as 2008-2021 not 2004-2021?

Reply:
Yes, this was intentional. I updated the PR description to reflect this - some of the source code in traitsui has Riverbank Copyright headers. We should not remove those headers - and we can claim copyright to those source code modules from the year after which Riverbank copyright.